lib/pyfrc/physics/core.py

```diff
@@ -272,54 +279,54 @@ def vector_drive(self, vx, vy, vw, tm_diff):
            :param vw: Clockwise rotational speed in rad/s
            :param tm_diff:         Amount of time speed was traveled
         '''
-        
+
         # if the robot is disabled, don't do anything
         if not self.robot_enabled:
             return
-        
+
         angle = vw * tm_diff
         vx = (vx * tm_diff)
         vy = (vy * tm_diff)
 
-        x = vx*math.sin(angle) + vy*math.cos(angle)
-        y = vx*math.cos(angle) + vy*math.sin(angle)
-        
+        x = vx * math.sin(angle) + vy * math.cos(angle)
+        y = vx * math.cos(angle) + vy * math.sin(angle)
```

Comment:
It never occurred to me that yapf would be so aggressive about applying arbitrary rules. The original formatting of these two lines are explicitly allowed and encouraged by PEP 8.

Reply:
I could go either way on this particular one because the plus sign is there.